docs/wiki/research/scaling/eip-4844.md

```diff
@@ -0,0 +1,220 @@
+
+### Changes on the Consensus Specs
+
+The upgrade which will introduce EIP-4844 into the Consensus Layer has been labeled **Deneb**.
+
+#### 1. Consensus Layer Validation
+
+As stated in the [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844#consensus-layer-validation) blobs are referenced but not fully encoded inside the beacon block body. Instead of embedding the full contents in the body, the blobs are propagated separately, as "sidecars".
+
+This "sidecars" design provides forward compatibility for further data increases by black-boxing `is_data_available()`, which means that in future upgrades `is_data_available()` implementation may change, for example to include data-availability-sampling (DAS), and the system interacting with this function is not required to understand the inner implementation of the function (in programming terms, `is_data_available()` can be considered as an interface method where the implementation may change over time).
+
+To learn more about the changes, see the consensus-specs:
+
+- [is_data_available](https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/fork-choice.md#is_data_available)
+- [BeaconBlockBody](https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/beacon-chain.md#beaconblockbody)
+- [p2p network](https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/p2p-interface.md#why-are-blobs-relayed-as-a-sidecar-separate-from-beacon-blocks)
+
+#### 2. Inclusion of KZG Committment versioned hashes
+
+The Consensus Layer (CL, also called Beacon chain) calls the [`process_execution_payload`](https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/beacon-chain.md#modified-process_execution_payload) function when a new block payload is submitted to the chain. This function is responsible to perform some validity checks on the block's payload and then invoke the Execution Layer (EL) via the `verify_and_notify_new_payload` function.
+
+Once invoked, the EL will:
+
+- validate the block payload
+- execute transactions inside the block payload
+- update the state, which is the result of executing transactions
+- build the new block
+- return the new block to the CL
+
+With EIP-4844, the `process_execution_payload` adds the parameter `versioned_hashes` to be passed to the `verify_and_notify_new_payload` function.
+
+`versioned_hashes` is an array of [hashes](https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/beacon-chain.md#modified-process_execution_payload) for each blob of KZG Committment, which are a type of cryptographic commitment particularly useful for their efficiency in creating and verifying proofs of data availability and correctness.
+
+KZG commitments provide the ability to prove that specific pieces of data are included in the set without revealing the entire dataset. This is particularly useful for scalability solutions because it does not require for every node to store the whole blockchain to prove transactions correcteness.
```

Comment:
for further context, you might want add a link to some more resources about KZG or at least the mpt ceremony https://github.com/ethereum/kzg-ceremony

Some historical discussion 
https://ethresear.ch/t/arithmetic-hash-based-alternatives-to-kzg-for-proto-danksharding-eip-4844/
https://ethresear.ch/t/sharding-multi-party-computation-ceremony/11779

Reply:
Added resources links at [commit](https://github.com/eth-protocol-fellows/protocol-studies/pull/51/commits/b00c671f4aad6930bd3fb0eab5121e257a1b9968)